Partition (WBP)/Functions/V57033G.VT157033.fun.yml

```diff
@@ -308,214 +308,370 @@
     Add_Entry To_List(#wlTestResultExactBoolean)
     
     * Search <2>
-    #LF_Test_Data := '2012-12-12 12:12:12.121212'
+    #LF_Test_Data := '2012-12-12 12:12:12.12'
     Add_Entry To_List(#wLTestKey)
     * ***Generic DateTime Results
-    #LF_Test_Result_L157033FDT := 0
-    #LF_Test_Result_L157033FD #LF_Test_Result_L157033FT := 10
+    If (#LF_Remote_Platform *EQ I)
+    #LF_Test_Result_L157033FDT #LF_Test_Result_L157033FD #LF_Test_Result_L157033FT := 0
```

Comment:
> This is wrong. 0 records???? The same issue as was occurring with SQL Server. You have inserted data. It must find it.
> 
> I've investigated the issue and decided its a defect thats not under your control. The IBMi supports up to 12 decimal points of precision.
> 
> This may be left as it is.
> 
> But, it would have been good for you to mention this issue rather have me stumble upon it.

I shall remove the IBMi  'if' check in this case. Wouldn't be necessary  to test  for IBMi for all the search results?

Reply:
I haven't validated in IBMi yet